modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc

```diff
@@ -389,75 +389,7 @@ class NDB_Menu_Filter_Form_Conflict_Resolver extends NDB_Menu_Filter_Form
 
         return true;
     }
-    /**
-     * Returns a list of candidates, users, etc.
-     * Need override for when Examiner field is conflicting,
-     * need Examiner's full name
-     *
-     * @return array
-     * @access private
-     */
-    function _getFullList()
```

Comment:
what exactly was the original problem? this appears to be undoing a change i made that was necessary for CCNA